Sources/Subs-Compat.php

```diff
@@ -8097,7 +8097,7 @@ function getMembersOnlineStats(array $membersOnlineOptions): array
 	 */
 	function displayDebug(): void
 	{
-		SMF\Logging::displayDebug();
+		SMF\Debug\DebugUtils::displayDebug();
```

Comment:
I believe a similar change will be needed in Utils::obExit() as well.

Reply:
commit amended. I seem to have missed copying that file over from the document root to the working tree.